stable/sumologic-fluentd/values.yaml

```diff
@@ -4,6 +4,21 @@ image:
   tag: latest
   pullPolicy: Always
 
+daemonset:
```

Comment:
Is anything gained by creating an intermediate `daemonset` object? Recommend making these top-level values.